aptos-move/framework/aptos-framework/sources/keyless_account.move

```diff
@@ -31,7 +44,7 @@ module aptos_framework::keyless_account {
     }
 
     #[resource_group_member(group = aptos_framework::keyless_account::Group)]
-    struct Configuration has key, store {
+    struct Configuration has key, store, drop, copy {
```

Comment:
i think an initialization func for this resource is missing

Reply:
i guess `update_configuration()` played the role before but not any more.

Reply:
actually not an issue because `on_new_epoch` plays this role now.

Reply:
hmm but you will still need an `initialize()` for genesis...

Reply:
Indeed. The genesis initialization will merely queue up the changes...